rosdep/base.yaml

```diff
@@ -3853,6 +3853,11 @@ smartmontools:
   debian: [smartmontools]
   fedora: [smartmontools]
   ubuntu: [smartmontools]
+smbclient:
+  arch: [smbclient]
+  debian: [smbclient]
+  fedora: [samba-client]
```

Comment:
this package doesn't seem to be available on fedora official package database: https://admin.fedoraproject.org/pkgdb/packages/samba*/